test/test_shellcheck.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+import testinfra
+
+run_local = testinfra.get_backend(
+    "local://"
+).get_module("Command").run
+
+def test_scripts_pass_shellcheck():
+    ''' Make sure shellcheck does not find anything wrong with our shell scripts '''
+    shellcheck = "find . -name 'update.sh' | while read file; do shellcheck \"$file\"; done;"
```

Comment:
Meant to put that last comment on this line, I put it on the output of py.test instead...

the fact travis is passing and my local build was failing is still weird...we may want to intentionally fail shellcheck to confirm it detects red builds properly.